frame/support/src/weights/extrinsic_weights.rs

```diff
@@ -15,32 +15,61 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-pub mod constants {
-	use frame_support::{
-		parameter_types,
-		weights::{constants, Weight},
-	};
-
-	parameter_types! {
-		/// Executing a NO-OP `System::remarks` Extrinsic.
-		pub const ExtrinsicBaseWeight: Weight = 125_000 * constants::WEIGHT_PER_NANOS;
-	}
+//! THIS FILE WAS AUTO-GENERATED USING THE SUBSTRATE BENCHMARK CLI VERSION 4.0.0-dev
+//! DATE: 2022-05-21 (Y/M/D)
+//!
+//! SHORT-NAME: `extrinsic`, LONG-NAME: `ExtrinsicBase`, RUNTIME: `Development`
+//! WARMUPS: `10`, REPEAT: `100`
+//! WEIGHT-PATH: `./frame/support/src/weights/`
+//! WEIGHT-METRIC: `Average`, WEIGHT-MUL: `1`, WEIGHT-ADD: `0`
+
+// Executed Command:
+//   ./target/production/substrate
+//   benchmark
+//   overhead
+//   --chain=dev
+//   --execution=wasm
+//   --wasm-execution=compiled
+//   --weight-path=./frame/support/src/weights/
+//   --warmup=10
+//   --repeat=100
+
+use frame_support::{
+	parameter_types,
+	weights::{constants::WEIGHT_PER_NANOS, Weight},
+};
+
+parameter_types! {
+	/// Time to execute a NO-OP extrinsic, for example `System::remark`.
+	/// Calculated by multiplying the *Average* with `1` and adding `0`.
+	///
+	/// Stats [NS]:
+	///   Min, Max: 85_600, 86_856
+	///   Average:  85_811
+	///   Median:   85_790
+	///   Std-Dev:  151.94
+	///
+	/// Percentiles [NS]:
+	///   99th: 86_196
+	///   95th: 85_992
+	///   75th: 85_853
+	pub const ExtrinsicBaseWeight: Weight = 85_811 * WEIGHT_PER_NANOS;
```

Comment:
This is the first time that we run this benchmark in Substrate.  
There is no real need for it since we have dedicated constants in Polkadot for the [Block](https://github.com/paritytech/polkadot/blob/297572bc325a97f5b0313ebe278841cd9b21c072/runtime/polkadot/constants/src/weights/block_weights.rs#L56) and [Extrinsic](https://github.com/paritytech/polkadot/blob/297572bc325a97f5b0313ebe278841cd9b21c072/runtime/polkadot/constants/src/weights/extrinsic_weights.rs#L56) weights.